plugins/inventory/nmap.py

```diff
@@ -34,6 +34,13 @@
             description: list of addresses to exclude
             type: list
             elements: string
+        port:
```

Comment:
Hmm, just one final thought: should this be called `ports` or `port`? The man page says `-p <port ranges>: Only scan specified ports`, so I think `ports` could be better suited. What do you think?

(Besides this, LGTM :) )

Reply:
I would say `ports` would be fine, but it is already being used for the `-sP` skip ports option.  Maybe that should have been `skip_ports`.  But if not `port` I would say `port_range` maybe instead of `port`.

Reply:
Ah, I wasn't aware of that option already being use. I agree, it should have better been called something else. Then let's keep `port` :)